DigitalLearningSolutions.Web/Views/Shared/Components/TextInput/Default.cshtml

```diff
@@ -1,30 +1 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
-@model TextInputViewModel
-
-<div class="nhsuk-form-group @(Model.HasError ? "nhsuk-form-group--error" : "")">
-  <label class="nhsuk-label" for="@Model.Name">
-    @Model.Label
-  </label>
-  @if (Model.HintText != null) {
-    <div class="nhsuk-hint" id="@Model.Name-hint">
-      @Model.HintText
-    </div>
-  }
-  @if (Model.HasError) {
-    <div class="nhsuk-u-padding-top-1 nhsuk-u-padding-bottom-3">
-      @foreach (var errorMessage in Model.ErrorMessages) {
-        <span class="error-message--margin-bottom-1 nhsuk-error-message">
-          <span class="nhsuk-u-visually-hidden">Error:</span> @errorMessage
-        </span>
-      }
-    </div>
-  }
-  <input class="nhsuk-input @Model.Class @(Model.HasError ? "nhsuk-input--error" : "")"
-         id="@Model.Id"
-         name="@Model.Name"
-         value="@Model.Value"
-         type="@Model.Type"
-         spellcheck="@Model.SpellCheck"
-         autocomplete="@Model.Autocomplete"
-         aria-describedby="@Model.Name-error @Model.Name-hint" />
-</div>
+﻿<partial name="~/Views/Shared/_TextInputItem.cshtml" model="@Model" />

```

Comment:
Rather than having this as a partial, would it make sense to have the DictionaryTextInput call the TextInput component in its view?

Reply:
This was how @SteveJacksonSoft suggested I do it

Reply:
Cool no worries then